infra/build-project.js

```diff
@@ -18,3 +18,17 @@ if (!isMainThread) {
     });
   }
 }
+
+// Helpers
+function runCmd(cmd, retryAttempts = 1) {
+  try {
+    return execSync(cmd, { stdio: 'pipe' });
+  } catch (err) {
+    if (retryAttempts > 0) {
+      console.log(`Retrying command '${cmd}'...`);
+      return runCmd(cmd, --retryAttempts);
```

Comment:
```suggestion
      return runCmd(cmd, retryAttempts - 1);
```

Reply:
:cry:

Reply:
Hey, it wasn't wrong 😉 I was just confused :-)

Reply:
I know :grin: